### PR TITLE
Automatically reload Electron instance on-file-change

### PR DESCRIPTION
**Pull request by mitchhentges**
Super-quick window reloads. Awesome.
Just `npm run development`, then away you go

### CODE REVIEW

**Comment by mitchhentges** [comment]
The console is a little noisy, especially when the window moves around, but I've created [an issue for electron-connect](https://github.com/Quramy/electron-connect/issues/27)

File: resource/index.html [hunk 1]
```diff
@@ -9,5 +9,9 @@
 <body>
     <div id="app"></div>
     <script src="./index.js"></script>
+    <!-- build:remove -->
+    <!-- Connect to server process -->
+    <script>require('electron-connect').client.create()</script>
```
Comment (nerdbeere) [inline resource/index.html]: What happens when electronConnect server is not started?

**Comment by nerdbeere** [comment]
Please merge master

File: resource/index.html (hunk 1 above)
Comment (mitchhentges) [inline resource/index.html]: It quietly fails in the console

**Comment by mitchhentges** [comment]
Master = merged :)

**Pull request merged**